.pylintrc

```diff
@@ -53,7 +47,7 @@ persistent=yes
 
 # Min Python version to use for version dependend checks. Will default to the
 # version used to run pylint.
-py-version=3.9
+py-version=3.7
```

Comment:
Does this mean we shouldnt use new python language features?

Reply:
Unfortunately yeah - with the SDK, we want to support as many customers as possible, so we have to stay more backwards-compatible with a broader range of python versions and other python dependencies than with our backend stuff.

On the other hand, I do think we might want limit to 3.8+. 3.7 is end-of-life and will not get security updates afetr June 2023 (https://endoflife.date/python).